Tasks/DownloadSecureFileV1/task.json

```diff
@@ -31,6 +31,14 @@
             "defaultValue": "",
             "required": true,
             "helpMarkDown": "Select the secure file to download to a temporary location on the agent. The file will be cleaned up after the pipeline runs."
+        },
+        {
+            "name": "retryCount",
+            "type": "string",
+            "label": "Retry Count",
+            "defaultValue": "5",
+            "required": "false",
+            "helpMarkDown": "Optional number of times to retry downloading a secure file if the download fails, defaults to five retries."
```

Comment:
nit: Is `defaults to five retries` necessary?

Reply:
No, probably not. I'll remove it, as I think a shorter description is better in this case.